### PR TITLE
[WIP] Fix notebooks and enable tests on notebooks

### LINKED ISSUE #10: Move notebooks to lightning-tutorials
## 🚀 Feature

better and faster testing of attached notebooks

### Additional context

https://blog.kovalevskyi.com/continues-integration-for-your-jupyter-notebooks-on-github-with-gcp-f72af5b08bcd

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
Fixes #10.

TODO
- [ ] Fixes notebooks (typo, URLs, formatting, installation instruction in notebooks)
- [x] Enable tests for notebooks on GPUs
- [x] Remove tests for notebooks on CPUs (GHA)

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/lightning-flash/tree/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? [not needed for typos/docs]
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-flash/blob/master/CHANGELOG.md)?

## PR review
 - [ ] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: flash_notebooks/custom_task_tutorial.ipynb [hunk 1]
```diff
@@ -13,7 +13,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Tutorial: Creating a Custom Task"
+    "# Creating a Custom Task"
```
Comment (akihironitta) [inline flash_notebooks/custom_task_tutorial.ipynb:16]:
```suggestion
    "# Tutorial: Creating a Custom Task"
```

**Comment by Borda** [comment]
Let's move also to running notebooks on GPU machine as it takes ages now...
cc: @ethanwharris

**Review by edgarriba — APPROVED** [review]

**Review by Borda — APPROVED** [review]

File: requirements/test.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 coverage
 codecov>=2.1
+nbval
```
Comment (justusschock) [inline requirements/test.txt:3]: This is never run?

Comment (akihironitta) [reply requirements/test.txt:3]: That's for pytest's `--nbval-lax` option here:
https://github.com/PyTorchLightning/lightning-flash/blob/2317782c5ef485523b386487ca1f55ef90b19804/.azure-pipelines/gpu-tests.yml#L61

**Review by justusschock — APPROVED** [review]

File: requirements/test.txt (hunk 2 above)
Comment (akihironitta) [reply requirements/test.txt:3]: That's for pytest's `--nbval-lax` option here:
https://github.com/PyTorchLightning/lightning-flash/blob/2317782c5ef485523b386487ca1f55ef90b19804/.azure-pipelines/gpu-tests.yml#L61

**Comment by Borda** [comment]
@akihironitta is it still WIP and how about your to-do in PR description?

**Comment by akihironitta** [comment]
@Borda It's still wip as some of the tests are failing... Let me have a look, or we can skip the failing tests for now if this is urgent.

**Comment by Borda** [comment]
> @Borda It's still wip as some of the tests are failing... Let me have a look, or we can skip the failing tests for now if this is urgent.

hehe seems like you got a wild card with all needed approvals before you finish it :D

**Pull request closed without merging**